DNN Platform/Admin Modules/Dnn.Modules.Console/ViewConsole.ascx.cs

```diff
@@ -1,539 +1,583 @@
-
-    using Dnn.Modules.Console.Components;
-    using DotNetNuke.Abstractions;
-    using DotNetNuke.Common.Utilities;
-    using DotNetNuke.Entities.Modules;
-    using DotNetNuke.Entities.Tabs;
-    using DotNetNuke.Entities.Users;
-    using DotNetNuke.Framework.JavaScriptLibraries;
-    using DotNetNuke.Instrumentation;
-    using DotNetNuke.Security.Permissions;
-    using DotNetNuke.Security.Roles;
-    using DotNetNuke.Services.Exceptions;
-    using DotNetNuke.Services.Localization;
-    using DotNetNuke.Services.Personalization;
+namespace Dnn.Modules.Console
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.IO;
+    using System.Linq;
+    using System.Text;
+    using System.Web.UI.WebControls;
+
+    using Dnn.Modules.Console.Components;
+    using DotNetNuke.Abstractions;
+    using DotNetNuke.Common.Utilities;
+    using DotNetNuke.Entities.Modules;
+    using DotNetNuke.Entities.Tabs;
+    using DotNetNuke.Entities.Users;
+    using DotNetNuke.Framework.JavaScriptLibraries;
+    using DotNetNuke.Instrumentation;
+    using DotNetNuke.Security.Permissions;
+    using DotNetNuke.Security.Roles;
+    using DotNetNuke.Services.Exceptions;
+    using DotNetNuke.Services.Localization;
+    using DotNetNuke.Services.Personalization;
     using DotNetNuke.Web.Client.ClientResourceManagement;
-    using Microsoft.Extensions.DependencyInjection;
-
-    public partial class ViewConsole : PortalModuleBase
-    {
-        private static readonly ILog Logger = LoggerSource.Instance.GetLogger(typeof(ViewConsole));
-        private readonly INavigationManager _navigationManager;
-        private ConsoleController _consoleCtrl;
-        private string _defaultSize = string.Empty;
-        private string _defaultView = string.Empty;
-        private int _groupTabID = -1;
-        private IList<TabInfo> _tabs;
-
-        public ViewConsole()
-        {
-            this._navigationManager = this.DependencyProvider.GetRequiredService<INavigationManager>();
-        }
-
-        public bool AllowSizeChange
-        {
-            get { return !this.Settings.ContainsKey("AllowSizeChange") || bool.Parse(this.Settings["AllowSizeChange"].ToString()); }
-        }
-
-        public bool AllowViewChange
-        {
-            get { return !this.Settings.ContainsKey("AllowViewChange") || bool.Parse(this.Settings["AllowViewChange"].ToString()); }
-        }
-
-        public bool IncludeHiddenPages
-        {
-            get { return this.Settings.ContainsKey("IncludeHiddenPages") && bool.Parse(this.Settings["IncludeHiddenPages"].ToString()); }
-        }
-
-        public ConsoleController ConsoleCtrl
-        {
-            get
-            {
-                if (this._consoleCtrl == null)
-                {
-                    this._consoleCtrl = new ConsoleController();
-                }
-
-                return this._consoleCtrl;
-            }
-        }
-
```

Comment:
Was this removed on purpose?

Reply:
Yes, nothing calls that and since it's not an extension point but a codebehind file, I thought it was ok to remove, but I can put it back if you think it was not a good idea...